h/static/styles/variables.scss

```diff
@@ -151,3 +151,15 @@ $score-width: 40px;
 $score-height: $score-width;
 $top-bar-height: 40px;
 $input-border-radius: 2px;
+
+// Variables for the new colour palette
```

Comment:
Should use American English spelling for 'color'.

One way we could iterate towards the new design would be to do a pass through the code which replaced all uses of the other grays with ones from this set.